### ISSUE TITLE
instructor resources for books?

### ISSUE THREAD

**Issue by meagansiu** [maltyeva/mindimp-pg#95]
this is a question for everybody, not just Maria :)

as we roll out the extended reading programs in the near future, we may find ourselves having to spend several hours a week teaching students about books we've never read before. i'm wondering if you guys would find it helpful if there was an instructor resources section for each book (or at least for the more complex ones), where I could post the links to the sources where I found discussion questions. often, these source also come with other writing activities and additional questions.

**Comment by ChasonMindImp** [comment]
I definitely think something like this is going to be necessary. And faster
overall, I think, than reading along with students, especially if we end up
juggling multiple ones who are in different programs and reading different
things at the same time.

On Fri, Apr 24, 2015 at 4:40 PM, meagansiu <notifications@github.com> wrote:

> this is a question for everybody, not just Maria :)
>
> as we roll out the extended reading programs in the near future, we may
> find ourselves having to spend several hours a week teaching students about
> books we've never read before. i'm wondering if you guys would find it
> helpful if there was an instructor resources section for each book (or at
> least for the more complex ones), where I could post the links to the
> sources where I found discussion questions. often, these source also come
> with other writing activities and additional questions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/95>.
>



-- 
Chason Dailey | 戴奇森
Program Manager and Advisor* | **思脉 **MindImp*
https://www.mindimp.com/

**Comment by jzthompson** [comment]
Yeah, I agree as well. An instructor resource section would be great. We could even add a link to sparknotes for that book or some other kind of summary of it. 

Another idea would be for instructors to gradually add brief summaries of each book assignment that they teach. Even just a couple of sentences on what occurred during that section of reading would probably be helpful. Of course, that does add some extra work, so just linking to sparknotes or some other summary might be the way to go.

**Comment by maltyeva** [comment]
Yeah, that should be definitely doable! 
What kinds of things would you guys want? 
Links/Text on the site/Attachments? :)

**Comment by jzthompson** [comment]
I think links and text would be enough. I'd envision the text being for instructor notes for each assignment, so it would be good admins could add separate text boxes for each week that a book has an assignment.
